runtime/ftplugin/mom.vim

```diff
@@ -0,0 +1,13 @@
+" Vim filetype plugin
+" Language:	roff(7)
```

Comment:
Language `roff`? I thought this is for `mom`?

Reply:
I beg your pardon, corrected